mlflow/models/evaluation/default_evaluator.py

```diff
@@ -1644,6 +1644,15 @@ def _log_eval_table(self):
         data = data.assign(**columns)
         artifact_file_name = f"{metric_prefix}{_EVAL_TABLE_FILE_NAME}"
         mlflow.log_table(data, artifact_file=artifact_file_name)
+        if self.eval_results_path:
+            eval_table_spark = self.spark_session.createDataFrame(data)
+            try:
+                eval_table_spark.write.mode(self.eval_results_mode).option(
+                    "mergeSchema", "true"
+                ).format("delta").saveAsTable(self.eval_results_path)
+            except Exception as e:
+                _logger.info(f"Saving eval table to delta table failed. Reason: {e}")
```

Comment:
Should we change this to logger.warn instead?

Reply:
i dont feel strongly either way, but figured that this might help with debugging during development.